sdk/communication/Azure.Communication.Chat/src/Models/CommunicationIdentifierModelKind.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This is a temporary solution.
+// Please remove when updated to a swagger version that supports 'kind' property.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Communication
+{
+    /// <summary> Type of CommunicationIdentifierModel. </summary>
+    public readonly partial struct CommunicationIdentifierModelKind : IEquatable<CommunicationIdentifierModelKind>
```

Comment:
would it be possible to keep this internal for chat for now? not to generate the API change...?

https://apiview.dev/Assemblies/Review/83a5714c2c6a4d03a9b671fe5148e9a1?diffRevisionId=17ce6284d9c844ab81b74d5261342ee4&diffOnly=False&revisionId=565e7b4109db46eda72443f540140776&doc=False

otherwise, we'd need an ARB review